apis/compute/v1alpha2/types.go

```diff
@@ -18,6 +18,11 @@ package v1alpha2
 
 import (
```

Comment:
Nitpick: these imports look misgrouped. Should be third party imports, then crossplane{-runtime} imports, then stack-aws imports.

Reply:
I think we need to fix this in the linter, as I'm not sure how much value is in manually fixing these